src/locale/pt-BR.json

```diff
@@ -22,6 +22,7 @@
     "housing-action-title": "Mobilização pelo Acesso à Moradia",
     "housing-action-description": "Onde <strong class='color-rent-strike'>ações de acesso à moradia</strong> estão ocorrendo ou ocorreram.",
     "housing-action-form": "Envie informações sobre ações pelo acesso à moradia de sua região (como ocupações, suspensão de alugueis, etc.)",
+    "submit": "Envie dados:",
     "resources": {
       "title": "",
       "description": "",
```

Comment:
line 28 is no longer needed, it must be deleted.

Reply:
Fixed!